realm/realm-library/src/androidTest/java/io/realm/RealmTests.java

```diff
@@ -2976,4 +2977,142 @@ public void clear_all() {
         assertEquals(0, realm.where(Cat.class).count());
         assertTrue(realm.isEmpty());
     }
+
+    @Test
+    public void waitForChange_emptyDataChange() throws InterruptedException {
+        final CountDownLatch bgRealmOpenedLatch = new CountDownLatch(1);
```

Comment:
No reason to have the suffix `latch` on these. That should be apparent from the type IMO.